src/components/LoginForm/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Fieldset, InputText, Button } from '@edx/paragon';
+import { FadeLoader } from 'react-spinners';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import login from '../../data/actions/loginForm';
+
+// TODO: redirect forgot password to correct site
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+    };
+    this.handleEmailChange = this.handleEmailChange.bind(this);
+    this.handlePasswordChange = this.handlePasswordChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleEmailChange(event) {
+    this.setState({
+      email: event.trim(),
+    });
+  }
+
+  handlePasswordChange(event) {
+    this.setState({
+      password: event.trim(),
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const { email, password } = this.state;
+    // used to send email and pass to api
+    this.props.login(email, password);
+    // clear form
+    this.state = {
+      email: '',
+      password: '',
+    };
+  }
+  render() {
+    const { isLoginPending } = this.props;
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <Fieldset
+          legend={<font color="grey">Login to edX from Enterprise</font>}
+          invalidMessage="This is invalid!"
+          variant={{ status: 'DANGER' }}
+          variantIconDescription="Error"
+        >
+          <InputText name="email" value={this.state.email} label="Email" type="text" onChange={(event) => { this.handleEmailChange(event); }} />
+          <InputText name="password" value={this.state.password} label="Password" type="password" onChange={(event) => { this.handlePasswordChange(event); }} />
+          <div className="form-addons">
+            <div className="submit-button">
```

Comment:
nit: Is the `submit-button` class name used anywhere else? If not, you could probably remove it.